### PR TITLE
support `ai dev new openai-asst-webpage --typescript`

### PR DESCRIPTION
**Pull request by robch**

### CODE REVIEW

File: src/ai/.x/templates/openai-asst-webpage-ts/_.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "_Language": "TypeScript",
+  "ClassName": "OpenAIAssistantsStreamingClass",
+  "AZURE_OPENAI_ASSISTANT_ID": "<insert your OpenAI assistant ID here>",
+  "AZURE_OPENAI_API_KEY": "<insert your Azure OpenAI API key here>",
+  "AZURE_OPENAI_API_VERSION": "<insert your Azure OpenAI API version here>",
+  "AZURE_OPENAI_ENDPOINT": "<insert your Azure OpenAI endpoint here>",
+  "AZURE_OPENAI_CHAT_DEPLOYMENT": "<insert your Azure OpenAI chat deployment name here>",
+  "OPENAI_API_KEY": "<insert your OpenAI API key here>",
+  "OPENAI_ORG_ID": "<insert your OpenAI organization ID here>",
+  "OPENAI_MODEL_NAME": "<insert your OpenAI model name here>"
```
Comment (manekinekko) [inline src/ai/.x/templates/openai-asst-webpage-ts/_.json:13]: I see there are other missing env vars:
- AZURE_OPENAI_CHAT_MODEL
- AZURE_OPENAI_EMBEDDING_DEPLOYMENT
- AZURE_OPENAI_EMBEDDING_MODEL
- AZURE_OPENAI_EVALUATION_DEPLOYMENT
- AZURE_OPENAI_EVALUATION_MODEL
- AZURE_OPENAI_KEY

File: src/ai/.x/templates/openai-asst-webpage-ts/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,581 @@
+import { marked } from "marked";
```
Comment (manekinekko) [inline src/ai/.x/templates/openai-asst-webpage-ts/src/index.ts:1]: There are a still some missing types in this file. Do you want me send a PR to add those?

File: src/ai/.x/templates/openai-asst-webpage-ts/_.json (hunk 1 above)
Comment (robch) [reply src/ai/.x/templates/openai-asst-webpage-ts/_.json:13]: i updated it to be symmetric with the other ones, and i removed the embedding and evalution vars as they're not needed in this level of the sample.

File: src/ai/.x/templates/openai-asst-webpage-ts/src/index.ts (hunk 2 above)
Comment (robch) [reply src/ai/.x/templates/openai-asst-webpage-ts/src/index.ts:1]: The .vscode files? i just removed that part of the README.md as the other "webpage" samples don't have that set up... Ifyou want to do a separate PR to add that, that'd be great.

**Pull request merged**